main.js

```diff
@@ -7,13 +7,29 @@ function cell(r, g, b, cid, f) {
 }
 
 function setupTable(t, f) {
-  for (var tr, i = 16; i < 256; i++) {
+  let tr;
+  let i;
+  for (i = 16; i < 232; i++) {
     if ((i - 16) % 36 === 0) {
       tr = $('<tr>');
       t.append(tr);
     }
     tr.append(cell(...getIndexedColor(i), i, f));
   }
+  // include all the greys for the ramp
+  let colors = [];
+  for (i = 16; i < 232; i += 43) {
+    colors.push([i, getIndexedColor(i)]);
+  }
+  for (i = 232; i < 256; i++) {
+    colors.push([i, getIndexedColor(i)]);
+  }
+  colors.sort((a, b) => a[1][0] < b[1][0]);
```

Comment:
It seems it has to be `-` here. Like that:
```javascript
colors.sort((a, b) => a[1][0] - b[1][0]);
// ---------------------------^
```
`<` wont work in my chrome 105

Reply:
Odd. I thought it would treat it like 0 or 1 even thought the method strictly wants <0, 0 or >0. That's an easy change, however, so fix coming!